### PR TITLE
(GH-124) Nexus Upgrade Documentation

### LINKED ISSUE #124: Add Nexus Update Procedure to QDE Docs
Need to take the docs we have written internally and create a new page under QDE>Setup>Nexus Upgrade.

### PR DESCRIPTION
**Pull request by ryanrichter94**
This PR adds documentation for upgrading Nexus inside a QDE instance. 

Fixes #124

### CODE REVIEW

File: input/en-us/quick-deployment/setup/upgrade-nexus-qde.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+Order: 40
+xref: v2-client-setup
```
Comment (steviecoaster) [inline input/en-us/quick-deployment/setup/upgrade-nexus-qde.md:3]: Not sure what we need to do here, but this _will_ need changes. Just not sure to _what_.

**Review by steviecoaster — CHANGES_REQUESTED** [review]

**Comment by gep13** [comment]
@ryanrichter94 @steviecoaster I have made a couple of minor corrections on this PR.

The main one (which was causing the build failure) was the updating of the xref.  This needs to be unique for each page.  This then allows Statiq to uniquely identify each page.

I also then updated the order to control "where" the page appears in the menu, and then updated the title to make it fit with the other pages in the section.

image

Let me know if you have any questions.

**Review by gep13 — APPROVED** [review]
LGTM!

**Comment by gep13** [comment]
@ryanrichter94 thanks for getting this added!  Your changes should be live on the site shortly.

**Pull request merged**